app/views/assets/_asset.html.erb

```diff
@@ -1,8 +1,31 @@
-<% gallery_view_id = nil unless defined? gallery_view_id %>
+<%
+unless defined? gallery_view_id
+  gallery_view_id = nil
+end
+
+unless defined? deletable
+  deletable = if asset.step
+                true
+              else
+                false
+              end
+end
+
+unless defined? editable
```

Comment:
Here the same. Isn't `editable` always undefined?

Reply:
It is also mostly for the future currently, we disabled edit options in the context menu in file preview.